libraries/AP_Motors/AP_MotorsHeli.cpp

```diff
@@ -603,3 +603,17 @@ void AP_MotorsHeli::update_turbine_start()
     }
 }
 
+bool AP_MotorsHeli::arming_checks(size_t buflen, char *buffer) const
+{
+    // run base class checks
+    if (!AP_Motors::arming_checks(buflen, buffer)) {
+        return false;
+    }
+
```

Comment:
Probably want to run a similar check that multi runs in it's arming check function